### DISCUSSION TITLE
Where are app logs (console.log) ?

### DISCUSSION THREAD

**Discussion by mariusa** [piku/piku#295] [Q&A] [answered]
Hi,

I'm used to heroku, where nodejs log, eg `console.log('test')`  shows in `heroku logs`

With local `piku logs` I only see
```
web.1    | gracefully (RE)spawned uWSGI master process (pid: 91980)
web.1    | [uwsgi-daemons] spawning "npm start" (uid: 1001 gid: 1001)
```
but the app's console.log don't show. Where do these go and how to see them from local?

Thanks

**Comment by rcarmo** [comment, marked as answer]
Check out the sample app - you need to merge stderr with stdout:

https://github.com/piku/sample-nodejs-app/blob/ba97c353a5747eb03345593df0f068041701abc6/Procfile#L1

**Reply by mariusa** [reply]
Thanks, that worked!
